tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go

```diff
@@ -0,0 +1,100 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+
+	tshelper "github.com/test-network-function/cnfcert-tests-verification/tests/networking/helper"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/networking/parameters"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+/*
+	The setup needs to be as mentioned here :
+	https://developers.redhat.com/articles/2021/08/27/using-virtual-functions-dpdk-red-hat-openshift
+*/
+
+var _ = Describe("Networking dpdk-cpu-pinning-exec-probe,", func() {
```

Comment:
Has this worked in a parallel execution?  We can put a `Serial` here if its not ready quite yet and then remove the `Serial` later.

Reply:
Initially I used with Serial when we had one pod deployment in default namespace. But now we moved to pod deployment in random namespace, hence parallel execution would work.